data/hai/unfettered side missions.txt

```diff
@@ -65,3 +65,472 @@ mission "Unfettered returning home"
+			branch "good or bad enough"
+				or
+					has "unfettered: beaten extra bugs"
+					"old reputation: Hai (Unfettered)" >= -2000
+			`	"Do not declare victory too soon, however, for we were not the only ones you had to beat," another adds.`
+			label "good or bad enough"
+			action
+				set "unfettered: beaten the bugs"
+	npc disable
+		government "Hai (Unfettered Challenger)"
+		to spawn
+			"total ships" == 1
+			has "flagship model: Anomalocaris"
+			not "unfettered: fled with the relic"
+			"old reputation: Hai (Unfettered)" < -2000
+		personality unconstrained entering plunders target heroic mute marked
+		ship "Sea Scorpion (Shredder)" "Requiem"
+		system "Hi Yahr"
+		conversation
+			branch "first one"
+				not "unfettered: beaten extra bugs"
+			`You have managed to defeat the last of the Unfettered challengers. Now it's time to land and claim your reward.`
+				goto score
+			label "first one"
+			`You receive no hail from this extra challenger, but there can be no doubt that they are indeed defeated.`
+			label score
+			action
+				set "unfettered: beaten extra bugs"
+	npc
+		government "Hai (Unfettered)"
+		to spawn
+			not "unfettered: fled with the relic"
+		ship "Lightning Bug (Unfettered Shipyards)"
+		personality launching staying surveillance marked
+		on disable
+			dialog `Mera hails you. "Stop! I am only here as an observer. If you destroy my ship, the deal is off!"`
+		on kill
+			fail "Hai Legacy 3"
+			set "unfettered: outcast"
+			dialog `As the Lightning Bug is destroyed you receive a hail from the local authorities. "You completely disrespected the rules of the duel and killed our most prominent researcher. You will not be welcome here anymore!"`
+	npc
+		government "Hai (Unfettered)"
+		to spawn
+			has "unfettered: fled with the relic"
+		personality unconstrained nemesis launching heroic marked
+		fleet
+			names "hai"
+			variant
+				"Sea Scorpion (Ionic)"
+	on visit
+		dialog "Either you have not defeated all of the Unfettered challengers in orbit yet, or you have broken the terms of their duel."
+	to complete
+		not "unfettered: fled with the relic"
+		has "unfettered: beaten the bugs"
+		has "flagship model: Anomalocaris"
+		or
+			"old reputation: Hai (Unfettered)" > -2000
+			has "unfettered: beaten extra bugs"
+	on complete
+		clear "unfettered: beaten the bugs"
+		clear "unfettered: beaten extra bugs"
+		clear "old reputation: Hai (Unfettered)"
+		log "Managed to use an old Anomalocaris to beat some Unfettered challengers, earning respect with the Unfettered Hai so long as the ship remains in your possession."
+		conversation
+			`When you land in one piece, Mera approaches, looking quite surprised. "You really won! At least it was close, right? Or did the captains make stories to justify their incompetence, like the cowards they are?"`
+			choice
```

Comment:
I think there should be another choice where you say something less positive about the duel, something like "That was close." or "I can't believe I won."